### DISCUSSION TITLE
how to use style and format together

### DISCUSSION THREAD

**Discussion by gmol** [charmbracelet/gum#120] [Q&A] [answered]
`gum format -t template '{{Color "120" "60" "Some Text"}}'` on its own  generates a text with colors.
`gum style --border double "$(gum format -t template '{{Color "120" "60" "Some Text"}}')"` generates 'Some Text' inside a frame but without colors.
How to combine them together?
OS: Ubuntu

**Comment by maaslalani** [comment, marked as answer]
Hey for you example you probably just want to use:

```
gum style --border double --foreground 120 --background 60 "Some Text"
```

For combining styles, you could use `gum style` in a subshell:

```
gum style --border double "$(gum style --foreground 120 --background 60 "Some Text")"
```
